modules/registry-securing-manually.adoc

```diff
@@ -16,30 +16,30 @@ Manually secure the registry to serve traffic via TLS:
 example:
 +
 ----
-$ podman-login registry.redhat.io
+$ podman login registry.redhat.io
 ----
 +
 . Fetch the service IP and port of the registry:
 +
 ----
-$ oc get svc/podman-registry
-NAME              LABELS                                    SELECTOR                  IP(S)            PORT(S)
-podman-registry   podman-registry=default                   podman-registry=default   172.30.124.220   5000/TCP
+$ oc get svc -n openshift-image-registry -o wide
+NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE   SELECTOR
+image-registry   ClusterIP   172.30.204.234   <none>        5000/TCP   25h   docker-registry=default
 ----
 +
 . You can use an existing server certificate, or create a key and server
 certificate valid for specified IPs and host names, signed by a specified CA. To
 create a server certificate for the registry service IP and the
-*podman-registry.default.svc.cluster.local* host name,
+`openshift-image-registry.default.svc.cluster.local` host name,
```

Comment:
The default route address will be like: default-route-openshift-image-registry.<cluster_name>, for example default-route-openshift-image-registry.apps.qe-521.qe.devcluster.openshift.com